pulser-core/pulser/waveforms.py

```diff
@@ -136,11 +140,10 @@ def draw(self, output_channel: Optional[Channel] = None) -> None:
         """
         fig, ax = plt.subplots()
         if not output_channel:
-            self._plot(ax, "rad/µs")
+            self._plot(ax)
```

Comment:
Eventually, the user could provide the units as an argument of this method ?

Reply:
Or maybe `ylabel`?